addons/test_mail_full/tests/test_rating.py

```diff
@@ -169,10 +171,69 @@ def test_rating_last_value_perfs(self):
 class TestRatingRoutes(HttpCase, TestRatingCommon):
 
     def test_open_rating_route(self):
+        """
+        _summary_
```

Comment:
Mmmh ? Is _summary_ a real thing ?

Reply:
Totally!
As in: my extension that prefills a docstring template when I type `"""` and gives me placeholders, which i *totally* did NOT leave in the commit!